web/static/js/components/user_list_item.jsx

```diff
@@ -8,18 +10,35 @@ import STAGES from "../configs/stages"
 
 const { VOTING } = STAGES
 
-export const UserListItem = ({ user, votes, stage }) => {
+export const UserListItem = ({ user, votes, stage, currentUser, actions }) => {
   const imgSrc = user.picture.replace("sz=50", "sz=200")
   const votesByUser = votes.filter(vote => vote.user_id === user.id).length
   const allVotesIn = votesByUser >= VOTE_LIMIT
+  const renderButton = !user.is_facilitator && currentUser.is_facilitator
```

Comment:
```suggestion
renderButton -> renderFacilitatorTransferButton 
```

or something that identifies the button's purpose in an expressive way